user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+.. _user-guide-getting-started-company-structure:
+
+Company Structure and Organization Selector
+===========================================
+
+Understanding the Company Structure
+-----------------------------------
+
+Each enterprise is structured in a meaningful way. Big enterprises usually have several branches/offices, each of which 
+may be divided into several major divisions comprised of departments that may be comprised of several functional groups, 
+etc. Even small companies organize their workforce into teams.
```

Comment:
Edit: Each enterprise is structured in a meaningful way. Big enterprises usually have several branches/offices, each of which may be divided into several major divisions and comprised of departments that may themselves be comprised of several other functional groups, etc. Even small companies organize their workforce into teams.